client/src/AppBundle/Resources/views/Report/Document/step1.html.twig

```diff
@@ -18,16 +20,26 @@
 
     {{ form_start(form, {attr: {novalidate: 'novalidate' }}) }}
 
+    <h2 class="govuk-heading-m">{{ (page ~ '.step1Heading') | trans }}</h2>
+
+    <div class="govuk-inset-text">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                {{ (page ~ '.selectHint1') | trans }}
+                <br>
+                <p>{{ (page ~ '.selectHint2') | trans }}</p>
```

Comment:
`<p>` is unnecessary now (`<br>` provides a newline, `<div>` provides the block-level element)
```suggestion
                {{ (page ~ '.selectHint2') | trans }}
```